system/languages/en.php

```diff
@@ -328,7 +328,7 @@
 		"art-block-resolved-subject" => "Blocking comment resolved: [[title]]",
 		"art-block-resolved-body" => "A blocking comment was resolved by [[author]] for:<br /><br /><a href=\"[[link]]\">[[title]]</a>",
 		"art-confirm-comment-subject" => "Please confirm post of your comment",
-		"art-confirm-comment-body" => "When you wish this comment to be published, you need to confirm by clicking the link underneath this line. This action is needed to avoid spam.<br /><a href=\"[[link]]\">[[link]]</a><br />On the website of [[sitename]] you (or someone that used your email address) have submitted a comment on the article: [[title]].<br /><br /> ------------------------------ <br />Your comment was:<br />[[comment]]<br />------------------------------<br />This is an automated email. Replying to this email is useless.",
+		"art-confirm-comment-body" => "De Stadsbron is a platform for contructive discussion, therefor we have formulated some rules that a commentor has to adhere to. Please consider if you comment adds new information or considerations to the discussion. Keep the tone civil. Make yourself known with your full name. We will complement your identity information if you do not make yourself known. We also reserve the right to remove your comment if it does not adhere to our rules. When you wish this comment to be published, you need to confirm by clicking the link underneath this line. This action is needed to avoid spam.<br /><a href=\"[[link]]\">[[link]]</a><br />On the website of [[sitename]] you (or someone that used your email address) have submitted a comment on the article: [[title]].<br /><br /> ------------------------------ <br />Your comment was:<br />[[comment]]<br />------------------------------<br />This is an automated email. Replying to this email is useless.",
```

Comment:
This is indeed a text that is too specific for Hypha. However, we could use the configured site title here, by adding `[[title]]` to the translation string, and by passing that title to the `__()` call that creates the translation (e.g. `__('art-confirm-comment-body', ['title' => hypha_getTitle()])`).